gadget/validate.go

```diff
@@ -179,6 +212,9 @@ var (
 		ubuntuSeedLabel,
 		ubuntuDataLabel,
 	}
+
+	// valid names for volumes under an eMMC schema
+	valideMMCVolumeNames = []string{"boot0", "boot1"}
```

Comment:
Sorry for being annoying here, I know @bboozzoo has a different opinion but `valideMMCVolumeNames` is not consistent with the names around like `validateEMMCStructureName` and tbh I prefer a capital E so you can visually separate words.